skll/config.py

```diff
@@ -485,11 +485,18 @@ def _parse_config_file(config_path):
     # how many folds should we run in parallel for grid search
     grid_search_folds = config.getint("Tuning", "grid_search_folds")
 
-    # what is the objective function for the grid search?
-    grid_objective = config.get("Tuning", "objective")
-    if grid_objective not in SCORERS:
-        raise ValueError('Invalid grid objective function: {}'
-                         .format(grid_objective))
+    # what are the objective functions for the grid search?
+    grid_objectives_string = config.get("Tuning", "objective")
+    grid_objectives = yaml.load(_fix_json(grid_objectives_string))
+    if isinstance(grid_objectives, string_types):
+        grid_objectives = [grid_objectives]
+    elif not isinstance(grid_objectives, list):
+        raise ValueError("objective should be a string or "
+                         "list of objectives")
+
+    if not all([(True if objective in SCORERS else False) for objective in grid_objectives]):
```

Comment:
Do we really need to be explicit with `True` and `False`? Can't you just say `if not all([objective in SCORES for objective in grid_objectives])' ?